__tests__/app.test.js

```diff
@@ -141,4 +141,37 @@ describe('app', () => {
         });
     });
   });
+  describe.only('POST /api/reviews/:review_id/comments', () => {
+    test('201: should return the posted comment', () => {
+      const requestBody = {
+        username: 'philippaclaire9',
+        body: 'Bad game',
+      };
+      return request(app)
+        .post('/api/reviews/5/comments')
+        .send(requestBody)
+        .expect(201)
+        .then(({body}) => {
+          expect(body.comment.author).toBe('philippaclaire9');
+          expect(body.comment.body).toBe('Bad game');
+          expect(body.comment.review_id).toBe(5);
+          expect(body.comment).toHaveProperty('comment_id', expect.any(Number));
+          expect(body.comment).toHaveProperty('votes', expect.any(Number));
+          expect(body.comment).toHaveProperty('created_at', expect.any(String));
+        });
+    });
+    test('400: should return an error when field is missing an entry', () => {
+      const requestBody = {
+        username: '',
+        body: 'Bad game',
+      };
+      return request(app)
+      .post('/api/reviews/10/comments')
+      .send(requestBody)
+      .expect(400)
+      .then(({body}) => {
+        expect(body.msg).toBe('Bad Request')
+      })
+    });
+  });
```

Comment:
Missing tests: 

Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 404, username does not exist { username: "jess", body: "validBody" }